src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const FileTypes = {
+    filters: [
+        { name: "video", type: "upload", extensions: [ "mp4", "webm", "mkv" ] },
```

Comment:
The binding between video/image (file type) and upload/image (Imgur API endpoint) shouldn't be in the constants file, rather in the Imgur uploader itself.